runsc/boot/fs.go

```diff
@@ -290,6 +291,11 @@ func adjustDirentCache(k *kernel.Kernel) error {
 	return nil
 }
 
+func configPageCache(args Args) {
```

Comment:
Globals like this are discouraged. I think if you were to move the LRU mechanism into pgalloc, we'd simply want to create a pgalloc.Options structure or something similar that would allow us to pass a richer configuration than is passed today.